Makefile

```diff
@@ -6,14 +6,21 @@ clean:
 	rm -f preview-extensions-ggqizro707
 	rm -f extensions.zip
 
-dist: clean
+dist-x86_64: clean
 	env GOARCH=amd64 GOOS=linux go build -ldflags="-s -w" -o ./extensions/newrelic-lambda-extension
 	touch preview-extensions-ggqizro707
 
-zip: dist
+dist-arm64: clean
+	env GOARCH=arm64 GOOS=linux go build -ldflags="-s -w" -o ./extensions/newrelic-lambda-extension
+	touch preview-extensions-ggqizro707
+
+zip-x86_64: dist-x86_64
+	zip -r extensions.zip preview-extensions-ggqizro707 extensions
+
+zip-arm64: dist-arm64
 	zip -r extensions.zip preview-extensions-ggqizro707 extensions
 
-publish: zip
+publish: zip-x86_64
 	aws lambda publish-layer-version --no-cli-pager --layer-name newrelic-lambda-extension --zip-file fileb://extensions.zip
```

Comment:
We'll need a different layer name for the arm64 version as well, wouldn't we?

Reply:
This isn't the actual layer publishing. This was just something I believe @MattWhelan added for local testing.